### PR TITLE
Add ECS Scheduled tasks to Terraform

### PR DESCRIPTION
**Pull request by kenoir**
### What is this PR trying to achieve?

Describing all our infra in code in order that it can be audited and recreated.

### Who is this change for?

💻 

### Have the following been considered/are they needed?

- [X] Run `terraform apply`.

### CODE REVIEW

**Comment by alexwlchan** [comment]
Resolves #806.

File: terraform/cloudwatch.tf [hunk 1]
```diff
@@ -47,3 +47,45 @@ resource "aws_cloudwatch_event_rule" "ecs_container_instance_state_change" {
 }
 PATTERN
 }
+
+# ECS Scheduled tasks
```
Comment (alexwlchan) [inline terraform/cloudwatch.tf]: I’d be tempted to put these in a separate file.

File: terraform/cloudwatch.tf [hunk 2]
```diff
@@ -47,3 +47,45 @@ resource "aws_cloudwatch_event_rule" "ecs_container_instance_state_change" {
...
+    task_definition_arn = "${module.gatling_catalogue_api.task_arn}"
+  }
+}
+
+resource "aws_iam_role" "scheduled_tasks_role" {
+  name               = "scheduled_tasks_role"
+  assume_role_policy = "${data.aws_iam_policy_document.assume_scheduled_tasks_role.json}"
+}
+
+data "aws_iam_policy_document" "assume_scheduled_tasks_role" {
```
Comment (alexwlchan) [inline terraform/cloudwatch.tf]: These should be in a separate file, or maybe a `scheduled_task` module?

You pass in a rule, a cluster ARN and a task definition ARN, rest is filled in for you.

Comment (kenoir) [inline terraform/cloudwatch.tf]: I wasn't sure about the best way to abstract these. A `scheduled_task` module makes sense I think.

File: terraform/ecs_script_task/outputs.tf [hunk 3]
```diff
@@ -0,0 +1,3 @@
+output "task_arn" {
+  value = "${aws_ecs_task_definition.task.arn}"
+}
```
Comment (alexwlchan) [inline terraform/ecs_script_task/outputs.tf]: Do we need this output? I don’t see it used anywhere.

**Pull request merged**